app/controllers/custom_mails_controller.rb

```diff
@@ -54,7 +54,17 @@ def send_mail
     render html: "The mails were queued for sending!"
   end
 
+  def send_mail_self
+    @mail.sent = false
```

Comment:
Why is this required?

Reply:
@ankitkataria `self_mail_self` is the route handler. I don't think it is suitable to reuse the batch email handler because it's a different route. 

As for the `@mail.sent = false`, this sending email to self is more of a preview.

I hope this addresses your concerns, and please ask more questions if m clarification is inadequate.

Thank you!

Reply:
> As for the @mail.sent = false, sending email to self is more of a preview, rather than sending the email out to the public.

It will be false be default. We can remove this from here.

Reply:
Ok, I will remove that line.